Makefile

```diff
@@ -106,6 +106,7 @@ shell: build-dirs build-image
 		-v "$$(pwd)/.go/pkg:/go/pkg:delegated" \
 		-v "$$(pwd)/.go/std:/go/std:delegated" \
 		-v "$$(pwd):/go/src/$(PKG):delegated" \
+		-v "$$(pwd)/vendor/k8s.io/api:/go/src/k8s.io/api:delegated" \
```

Comment:
I added this already (https://github.com/skriss/ark/pull/1/files#diff-b67911656ef5d18c4ae36cb6741b7965R105) so can remove this one